src/deepmol/datasets/datasets.py

```diff
@@ -554,7 +571,7 @@ def select(self, indexes: List[int], axis: int = 0):
             self.features2keep = np.array(list(set(self.features2keep) - set(indexes_to_delete)))
             self.features2keep = np.sort(self.features2keep)
 
-    def merge(self, datasets: List[Dataset]) -> 'NumpyDataset':
+    def merge(self, datasets: List[Dataset]):
```

Comment:
It returns a NumpyDataset, right? Why the removal of the returned type?

Reply:
Yes, it does. I removed it because we were not doing it for any other methods... I will add it to all.